zcash_client_backend/src/fees.rs

```diff
@@ -301,6 +330,51 @@ impl Default for DustOutputPolicy {
     }
 }
 
+/// `EphemeralParameters` can be used to specify variations on how balance
+/// and fees are computed that are relevant to transactions using ephemeral
+/// transparent outputs.
+#[cfg(feature = "transparent-inputs")]
+#[derive(Clone, Debug, Default, PartialEq, Eq)]
```

Comment:
Nit: I don't like the `Default` trait; I think it shouldn't exist and we shouldn't use it.

Reply:
We can use `EphemeralParameters::default()` rather than `Default::default()` if you prefer. I didn't want to make it `Optional<EphemeralParameters` because that just adds boilerplate, and one of the main aims of the refactoring that replaced multiple function parameters with `EphemeralParameters` was callers should have a way to specify the default that isn't going to change.

Reply:
Use of the `Default` trait was removed.